src/main/java/appeng/recipes/RecipeHandler.java

```diff
@@ -74,696 +31,15 @@
-		{
-			throw new RecipeException( "Not applicable for website" );
-		}
-
-		final IDefinitions definitions = AEApi.instance().definitions();
-		final IItems items = definitions.items();
-		final IBlocks blocks = definitions.blocks();
-
-		final Optional<Item> maybeCrystalSeedItem = items.crystalSeed().maybeItem();
-		final Optional<Item> maybeSkyStoneItem = blocks.skyStoneBlock().maybeItem();
-		final Optional<Item> maybeCStorageItem = blocks.craftingStorage1k().maybeItem();
-		final Optional<Item> maybeCUnitItem = blocks.craftingUnit().maybeItem();
-		final Optional<Item> maybeSkyChestItem = blocks.skyStoneChest().maybeItem();
-
-		if( maybeCrystalSeedItem.isPresent() && is.getItem() == maybeCrystalSeedItem.get() )
-		{
-			final int dmg = is.getItemDamage();
-			if( dmg < ItemCrystalSeed.NETHER )
-			{
-				realName += ".Certus";
-			}
-			else if( dmg < ItemCrystalSeed.FLUIX )
-			{
-				realName += ".Nether";
-			}
-			else if( dmg < ItemCrystalSeed.FINAL_STAGE )
-			{
-				realName += ".Fluix";
-			}
-		}
-		else if( maybeSkyStoneItem.isPresent() && is.getItem() == maybeSkyStoneItem.get() )
-		{
-			switch( is.getItemDamage() )
-			{
-				case 1:
-					realName += ".Block";
-					break;
-				case 2:
-					realName += ".Brick";
-					break;
-				case 3:
-					realName += ".SmallBrick";
-					break;
-				default:
-			}
-		}
-		else if( maybeCStorageItem.isPresent() && is.getItem() == maybeCStorageItem.get() )
-		{
-			switch( is.getItemDamage() )
-			{
-				case 1:
-					realName += "4k";
-					break;
-				case 2:
-					realName += "16k";
-					break;
-				case 3:
-					realName += "64k";
-					break;
-				default:
-			}
-		}
-		else if( maybeCUnitItem.isPresent() && is.getItem() == maybeCUnitItem.get() )
-		{
-			switch( is.getItemDamage() )
-			{
-				case 1:
-					realName = realName.replace( "Unit", "Accelerator" );
-					break;
-				default:
-			}
-		}
-		else if( maybeSkyChestItem.isPresent() && is.getItem() == maybeSkyChestItem.get() )
-		{
-			switch( is.getItemDamage() )
-			{
-				case 1:
-					realName += ".Block";
-					break;
-				default:
-			}
-		}
-		else if( is.getItem() instanceof ItemMaterial )
-		{
-			realName = realName.replace( "ItemMultiMaterial", "ItemMaterial" );
-			realName += '.' + ( (ItemMaterial) is.getItem() ).getTypeByStack( is ).name();
-		}
-		else if( is.getItem() instanceof ItemPart )
-		{
-			realName = realName.replace( "ItemPart", "ItemPart" );
-			realName += '.' + ( (ItemPart) is.getItem() ).getTypeByStack( is ).name();
-		}
-		else if( is.getItemDamage() > 0 )
-		{
-			realName += "." + is.getItemDamage();
-		}
-
-		return realName;
-	}
-
-	String alias( final String in )
-	{
-		Preconditions.checkNotNull( in );
-
-		final String out = this.data.aliases.get( in );
-
-		if( out != null )
-		{
-			return out;
-		}
-
-		return in;
-	}
-
 	@Override
 	public void parseRecipes( final IRecipeLoader loader, final String path )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "IRecipeLoader"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)